vassal-app/src/main/java/VASSAL/build/module/turn/TurnTracker.java

```diff
@@ -847,8 +897,9 @@ protected void setControls() {
     private static final long serialVersionUID = 1L;
     private IconButton nextButton;
     private IconButton prevButton;
+    private boolean initialized = false;
 
-    protected final int BUTTON_SIZE = 22; //NOPMD
+    protected final int BUTTON_SIZE = 22; @Deprecated //NOPMD
```

Comment:
Where's that `@Deprecated` supposed to be?

Reply:
Oh just that BUTTON_SIZE isn't actually used for anything. Should I just get rid of that line? Can I?

Reply:
No, I just mean it's in a strange place. If you want to deprecate `BUTTON_SIZE`, it should be on the line _before_ the declaration.

Reply:
Ah, gotcha -- fixed